core/src/main/scala/cromwell/core/KnowsWhatTimeItIs.scala

```diff
@@ -1,9 +1,15 @@
 package cromwell.core
 
 import java.sql.Timestamp
-import java.util.Date
+import java.time.OffsetDateTime
+import org.joda.time.DateTime
 
+object KnowsWhatTimeItIs {
+  implicit class JodaTimestampFormatter(val underlying: Timestamp) extends AnyVal {
+    def asJodaString = new DateTime(underlying).toString
```

Comment:
You can kill the use of the [deprecated](http://www.joda.org/joda-time/) Java 7- joda.time by using the Java 8+ java.time equivalents:

```scala
import java.time.format._
def asJodaStringEquivalent = underlying.
  toLocalDateTime.
  atZone(ZoneId.systemDefault).
  format(DateTimeFormatter.ISO_OFFSET_DATE_TIME)
```